drivers/video/ov2640_regs.h

```diff
@@ -0,0 +1,410 @@
+/*
```

Comment:
AFAIU, you can put all this content in .c file directly, no gain in having a .h if nothing is shared or exported.

Reply:
Right, merged them.